src/signals/settings/users/containers/Overview/index.js

```diff
@@ -31,11 +32,18 @@ const HeaderButton = styled(Button)`
   }
 `;
 
+const StyledSearchbar = styled(SearchBar)`
+  > button {
+    display: none;
+  }
+`;
+
 export const UsersOverviewContainer = ({ pageSize, userCan }) => {
   const history = useHistory();
   const { pageNum } = useParams();
   const [page, setPage] = useState(1);
-  const { isLoading, users: { list: data }, users } = useFetchUsers({ page, pageSize });
+  const [filters, setFilters] = useState({});
+  const { isLoading, users: { list: data }, users } = useFetchUsers({ page, pageSize, filters });
```

Comment:
This was the construct I was looking for !!!!!1!
Getting both the object as well as the the destructured object 🤦‍♂